src/audio_al.cpp

```diff
@@ -605,3 +607,5 @@ void ALAudio::SE_Stop() {
 	}
 	se_src_.clear();
 }
+
+#endif
```

Comment:
Also a policy to include or not include comments about closing preprocessor conditionals ` // HAVE_OPENAL`